api/agentpb/agent.proto

```diff
@@ -81,46 +82,49 @@ message SetStateResponse {
 
 }
 
-// ActionName defines allowed action names.
-enum ActionName {
-  ACTION_NAME_INVALID = 0;
-  PT_SUMMARY = 1;
-  PT_MYSQL_SUMMARY = 2;
-  MYSQL_EXPLAIN = 3;
-}
-
-// StartActionRequest represents request to run an action by name.
+// StartActionRequest is a ServerMessage asking pmm-agent to start action.
 message StartActionRequest {
-  string action_id = 1;
-  ActionName name = 2;
```

Comment:
Name is a unique identifier for Nodes and Service, similar to ID, but human-readable and -assignable.
This is type.

Reply:
- Nodes and Services are placed in different contexts, so we shouldn't define actions the same way as for nodes and services. Every time we are talking about things we should talk about this in some context.
- I called this `ActionName` as in my mind `type` is something more generalized, like `shell-action, sql-action, no-sql-action, etc...` those thing looks like types, but the current is not IMHO.

If you think the word `Type` describes this thing more clearly, ok, it's up to you. I don't think so.

Reply:
Sure. Can we not overload the words "type" and "name" in the context of PMM? Make "type" mean the same thing in Inventory and Management APIs.